openassessment/templates/openassessmentblock/peer/oa_peer_assessment.html

```diff
@@ -68,15 +68,23 @@ <h3 class="peer-assessment__display__title">
 
                                 {% include "openassessmentblock/oa_submission_answer.html" with answer=peer_submission.answer answer_text_label="Your peer's response to the question above:" %}
 
-                                {% if allow_file_upload and peer_file_url %}
+                                {% if file_upload_type and peer_file_url %}
                                 <header class="peer-assessment__display__header">
                                     <h3 class="peer-assessment__display__title">
-                                        {% trans "Associated Image" %}
+                                        {% trans "Associated Uploaded File" %}
```

Comment:
Nit: this doesn't look right. I'll have the documentation team take a pass at reviewing user-facing strings.